### PR TITLE
New feature #19191: Add destination survey id argument to copy_survey…

### PR DESCRIPTION
**Pull request by mfavetti**
… rpc function

New feature #19191: Add destination survey id argument to copy_survey rpc function

### CODE REVIEW

File: application/helpers/remotecontrol/remotecontrol_handle.php [hunk 1]
```diff
@@ -288,9 +288,12 @@ public function import_survey($sSessionKey, $sImportData, $sImportDataType, $sNe
      * @param string $sNewname name of the new survey
```
Comment (Shnoulle) [inline application/helpers/remotecontrol/remotecontrol_handle.php:288]: phpdoc

**Review by Shnoulle — APPROVED** [review]

**Review by Shnoulle — CHANGES_REQUESTED** [review]
phpdoc needed

File: application/helpers/remotecontrol/remotecontrol_handle.php (hunk 1 above)
Comment (mfavetti) [reply application/helpers/remotecontrol/remotecontrol_handle.php:288]: yes, thank you! i added it

**Comment by mfavetti** [comment]
> phpdoc needed

added

**Review by Shnoulle — APPROVED** [review]

**Comment by Shnoulle** [comment]
@olleharstedt new feature : need unit test ?

**Comment by olleharstedt** [comment]
Else just wait for @tiborpacalat to test it. Whatever comes first. :))

**Pull request merged**